addons/website_sale/views/templates.xml

```diff
@@ -53,6 +53,162 @@
+                                            <i class="fa fa-instagram o_editable_media rounded shadow-sm"/>
+                                        </a>
+                                        <a
+                                            href="/website/social/youtube"
+                                            class="s_social_media_youtube"
+                                            target="_blank"
+                                            aria-label="YouTube"
+                                        >
+                                            <i class="fa fa-youtube-play o_editable_media rounded shadow-sm"/>
+                                        </a>
+                                        <a
+                                            href="/website/social/tiktok"
+                                            class="s_social_media_tiktok"
+                                            target="_blank"
+                                            aria-label="TikTok"
+                                        >
+                                            <i class="fa fa-tiktok o_editable_media rounded shadow-sm"/>
+                                        </a>
+                                        <a
+                                            href="/website/social/twitter"
+                                            class="s_social_media_twitter"
+                                            aria-label="Twitter"
+                                        >
+                                            <i class="fa fa-twitter o_editable_media rounded shadow-sm"/>
+                                        </a>
+                                    </div>
+                                </div>
+
+                                <!-- Payment Icons -->
+                                <div class="row">
+                                    <h5>We accept</h5>
+                                    <div
+                                        class="s_supported_payment_methods o_not_editable"
+                                        data-snippet="s_supported_payment_methods"
+                                        data-name="Supported Payment Methods"
+                                        data-limit="8"
+                                        data-height="32px"
+                                        data-oe-protected="true"
+                                    />
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </section>
+            </div>
+        </div>
+
+        <xpath expr="//span[hasclass('o_footer_copyright_name')]" position="replace">
+            <span class="o_footer_copyright_name me-2">
+                &amp;copy;
+                <span t-field="res_company.name" itemprop="name">Company name</span>
+                 - <a href="#"><p class="d-none d-md-inline">Terms &amp;</p> Conditions</a>
```

Comment:
- span > a > p is not valid HTML
- This should not use a t-field, we removed it in website for a reason. We don't want user ediing the company name just by changing their copyright in their website. Also it does not allow to properly edit that section (using bold, italic, whatever).